packages/features/tips/UpgradeTip.tsx

```diff
@@ -29,8 +29,8 @@ export function UpgradeTip({
   const { isLoading, hasTeamPlan } = useHasTeamPlan();
 
   if (hasTeamPlan) return children;
-
-  if (isParentLoading || isLoading) return <>{isParentLoading}</>;
+  
+  if (isLoading) return <>{isParentLoading}</>;
```

Comment:
This is a new change. @saurabbot can you verify the other places as well where UpgradeTip is being used if not already,.